lib/jekyll/deprecator.rb

```diff
@@ -9,20 +9,23 @@ def process(args)
                           'serve' subcommand."
       arg_is_present? args, "--no-server", "To build Jekyll without launching a server, \
                           use the 'build' subcommand."
-      arg_is_present? args, "--auto", "The switch '--auto' has been replaced with '--watch'."
+      arg_is_present? args, "--auto", "The switch '--auto' has been replaced with \
+                          '--watch'."
       arg_is_present? args, "--no-auto", "To disable auto-replication, simply leave off \
                           the '--watch' switch."
       arg_is_present? args, "--pygments", "The 'pygments'settings has been removed in \
                           favour of 'highlighter'."
-      arg_is_present? args, "--paginate", "The 'paginate' setting can only be set in your \
+      arg_is_present? args, "--paginate", "The 'paginate' setting can only be set in \
+                          your config files."
+      arg_is_present? args, "--url", "The 'url' setting can only be set in your \
                           config files."
-      arg_is_present? args, "--url", "The 'url' setting can only be set in your config files."
       no_subcommand(args)
     end
 
     def no_subcommand(args)
-      if args.size > 0 && args.first =~ /^--/ && !%w(--help --version).include?(args.first)
-        deprecation_message "Jekyll now uses subcommands instead of just switches. Run `jekyll help` to find out more."
+      if args.!empty? && args.first =~ /^--/ && !%w(--help --version).include?(args.first)
+        deprecation_message "Jekyll now uses subcommands instead of just switches. \
+                          Run `jekyll help` to find out more."
```

Comment:
Close the quote to avoid having all that white space included in the message.

```ruby
        deprecation_message "Jekyll now uses subcommands instead of just switches. " \
            "Run `jekyll help` to find out more."
```